Src/PerformanceCollector/Unit.Tests/PerformanceCollectorTestBase.cs

```diff
@@ -8,22 +8,17 @@
     using Microsoft.VisualStudio.TestTools.UnitTesting;
 
     /// <summary>
-    /// PerformanceCollector tests.
+    /// PerformanceCollector test base.
     /// </summary>
-    [TestClass]
-    public class PerformanceCollectorTests
-    {
-        [TestMethod]
-        [TestCategory("RequiresPerformanceCounters")]
-        public void PerformanceCollectorSanityTest()
+    public class PerformanceCollectorTestBase
+    { 
+        internal void PerformanceCollectorSanityTest(IPerformanceCollector collector)
```

Comment:
I just made them as it complained about the argument having lower visibility than the method. They can be public. This is a base class that is used by two collectors, which is why I have moved the code out.